python/exercise2/time_window.py

```diff
@@ -4,10 +4,10 @@ class TimeWindow:
     """ Observations for a TIEGCM experiment """
     
     def __init__(self, start_time, end_time, delta):
-        """ Initializes the observations associated with a TIEGCM experiment
+        """ Initialize the window and model times associated with a TIEGCM experiment
         
-        start_time - string '%Y-%m-%d %H:%M:%S' model start time of experiment
-        end_time   - string '%Y-%m-%d %H:%M:%S' model end time of experiment
+        start_time - string '%Y-%m-%d %H:%M:%S' model inital time
+        end_time   - string '%Y-%m-%d %H:%M:%S' model end time
         delta - integer  assimilation window time in hours
```

Comment:
Might add delta limits here. If delta cannot be zero it should be changed to (0, 23] or [1, 23]